keps/sig-cli/5295-kyaml/kep.yaml

```diff
@@ -0,0 +1,48 @@
+title: KYAML
+kep-number: 5295
+authors:
+  - "@thockin"
+  - "@bentheelder"
+owning-sig: sig-cli
+participating-sigs:
+  - sig-docs
+  - sig-architecture
+  - sig-apps
+status: implementable #|implemented|deferred|rejected|withdrawn|replaced
+creation-date: 2025-05-09
+reviewers:
+  - "@lmktfy"
+approvers:
+  - "@soltysh"
+  - "@liggitt"
+
+see-also: []
+replaces: []
+
+# The target maturity stage in the current dev cycle for this KEP.
+# If the purpose of this KEP is to deprecate a user-visible feature
+# and a Deprecated feature gates are added, they should be deprecated|disabled|removed.
+stage: alpha #beta|stable
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.34"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.34"
+  #beta:
+  #stable:
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: KUBECTL_KYAML
+    components:
+      - kubectl
```

Comment:
Does anyone know if we're going to support enabling kubectl feature gates via [.kuberc](https://kubernetes.io/docs/reference/config-api/kuberc.v1alpha1/) ?

Reply:
Not planned, yet, but I'm not saying we won't add it in the near future.